releasenotes/notes/update-cs-csdg-in-equivalence-library-c1e70d3246f4aa6d.yaml

```
@@ -0,0 +1,7 @@
+---
+features_synthesis:
+  - |
+    Add more equivalences to :class:`.CSGate` and :class:`.CSdgGate` in the equivalence library.
+    Previously, the basis translator used 3 :class:`.CXGate` gates for the synthesis of
+    :class:`.CSdgGate` and now it uses only two :class:`.CXGate`.
+    For both gates we reduce the number of single-qubit gates in their basis translation.
```

A nitpick: this paragraph implicitly assumes that you are transpiling to the `["cx", "u"]` basis set, without explicitly mentioning this. Also using both "synthesis" and "translation" within the same sentence might be a bit confusing. Maybe something like this:

```suggestion
    The standard equivalence library has additional equivalences for :class:`.CSGate` and :class:`.CSdgGate`.
    When transpiling to the `["cx", "u"]` basis set, the :class:`.BasisTranslator` transpiler pass now uses
    2  :class:`.CXGate` gates instead of 3 for :class:`.CSdgGate`, and in addition reduces the number of single-qubit
    gates in both cases.
```